xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -1083,9 +1085,7 @@ void CRenderManager::PrepareNextRender()
   double frameOnScreen = m_dvdClock.GetClock();
   double frametime = 1.0 / g_graphicsContext.GetFPS() * DVD_TIME_BASE;
 
-  // correct display latency
-  // internal buffers of driver, assume that driver lets us go one frame in advance
-  double totalLatency = DVD_SEC_TO_TIME(m_displayLatency) - DVD_MSEC_TO_TIME(m_videoDelay) + 2* frametime;
```

Comment:
RM can call GraphicsContext, then gfx context can call windowing for the remaining info.

off topic:
I was thinking more about usage of g_windowing and I am inclined to drop direct use of this global in general. 
For platform specific functions implementations could call something like CGraphicsContext::GetWindowingInterface and do a dynamic cast in order to get a specific interface.